src/modules/core/components/ActionsList/ActionsListItem.tsx

```diff
@@ -93,6 +95,13 @@ const ActionsListItem = ({
     ({ ethDomainId }) => ethDomainId === parseInt(toDomainId, 10),
   );
 
+  const getFormattedRole = () => {
+    const roleNameMessage = { id: `role.${role}` };
+    return `${setTo ? 'Assign' : 'Remove'} ${formatMessage(
+      roleNameMessage,
+    ).toLowerCase()} permission`;
+  };
+
```

Comment:
As I've stated above, consider using two message descriptors for this event, and in that case, you can use this same function, to just return the correct message descriptor id